code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -947,7 +946,8 @@
 	user <<"<span class='warning'>[src] suddenly vanishes!</span>"

```

Comment:
Can you change this to to_chat while you are at it? I think my regex didn't detect `<<"`. There's another output in recoil() aswell.